Add bug-reports and dev-repo fields for most recent version of my packages; fix maintainer line for all my packages

darioteixeira opened the pull request:
Note: the fix of the maintainer line was motivated by the OPAM Weather Service reports.

dsheets commented on packages/lambdoc/lambdoc.1.0-beta3/opam:
```diff
@@ -1,31 +1,31 @@
+maintainer: "Dario Teixeira <dario.teixeira@nleyten.com>"
 authors: ["Dario Teixeira <dario.teixeira@nleyten.com>"]
-homepage: "http://lambdoc.forge.ocamlcore.org/"
+homepage: "https://lambdoc.forge.ocamlcore.org/"
```
This URL gives me a TLS error because the cert is only for forge.ocamlcore.org and ocamlcore.org.

dsheets commented:
`lambdoc.1.0-beta1` has a missing constraint on its `bookaml` dependency. It requires `"bookaml" {< "2.0"}` due to an interface change. Unfortunately, `bookaml.1.0` is the only version satisfying this constraint and it was removed in #3623. This makes `lambdoc.1.0-beta1` uninstallable.

darioteixeira commented:
@dsheets: I've removed the lambdoc.1.0-beta1, given its uninstallable (and outdated) status.  I've also switched the homepage URLs from HTTPS to HTTP.  Finally, I upped the version number and added a few of the missing fields to placate opam-lint.

darioteixeira commented:
Travis is complaining it cannot download OCamlnet.  Seems like a transient problem -- can someone restart it?

dsheets commented:
With so many modified packages, Travis was timing out. Almost all of the changes were purely metadata and do not affect building so merging.
